docs/quickstart.rst

```diff
@@ -61,198 +199,276 @@ instance had already existing in the destination directory, this command
-   df /mnt/node_a
-   Filesystem     1K-blocks   Used Available Use% Mounted on
-   tmpfs             409600 409600         0 100% /mnt/node_a
+It shows
 
-After a number of failed attempts to restart node A, its keeper signals
-that the node is unhealthy and the node is put into the "demoted" state.
-The monitor promotes node B to be the new primary.
+.. code-block::
 
-.. code-block:: bash
+    count
+  ---------
+   1000000
+
+Cause a node failure
+--------------------
 
-   pg_autoctl show state --pgdata ./monitor
-        Name |   Port | Group |  Node |     Current State |    Assigned State
-   ----------+--------+-------+-------+-------------------+------------------
-   127.0.0.1 |   6010 |     0 |     1 |           demoted |        catchingup
-   127.0.0.1 |   6011 |     0 |     2 |      wait_primary |      wait_primary
+This plot is too boring, time to introduce a problem. We’ll turn off VM for
+node B (currently the primary after our previous failover) and watch node A
+get promoted.
 
+In one terminal let’s keep an eye on events:
 
-Node B cannot be considered in full "primary" state since there is no
-secondary present. It is marked as "wait_primary" until a secondary
-appears.
+.. code-block:: bash
 
-A client, whether a web server or just psql, can list multiple
-hosts in its PostgreSQL connection string, and use the parameter
-``target_session_attrs`` to add rules about which server to choose.
+   watch -d \
+     ssh -l ha-admin `vm_ip monitor` pg_autoctl show state --pgdata monitor
 
-To discover the url to use in our case, the following command can be used:
+In another terminal we’ll turn off the virtual server.
 
 .. code-block:: bash
 
-   pg_autoctl show uri --formation default --pgdata ./monitor
-   postgres://127.0.0.1:6010,127.0.0.1:6011/?target_session_attrs=read-write
+   az vm stop \
+     --resource-group ha-demo \
+     --name ha-demo-b
 
-Here we ask to connect to either node A or B -- whichever supports reads and
-writes:
+After a number of failed attempts to talk to node B, the monitor determines
+the node is unhealthy and puts it into the "demoted" state.  The monitor
+promotes node A to be the new primary.
 
 .. code-block:: bash
 
-   psql \
-     'postgres://127.0.0.1:6010,127.0.0.1:6011/?target_session_attrs=read-write'
+   .
+                              Name |   Port | Group |  Node |     Current State |    Assigned State
+   --------------------------------+--------+-------+-------+-------------------+------------------
+   ha-demo-a.internal.cloudapp.net |   5432 |     0 |     1 |      wait_primary |      wait_primary
+   ha-demo-b.internal.cloudapp.net |   5432 |     0 |     2 |           demoted |        catchingup
+
+Node A cannot be considered in full "primary" state since there is no secondary
+present, but it can still serve client requests. It is marked as "wait_primary"
+until a secondary appears, to indicate that it's running without a backup.
```

Comment:
Without a secondary. I think using the term “backup” here would be rather confusing to the users. A secondary server can not be considered a backup: imagine running a `TRUNCATE` or a `DROP TABLE` on the primary, then by the time you would `switchover` to the other node the command has been replayed already. So let's not use the term “backup” anywhere here.